### PR TITLE
Structure

### PR DESCRIPTION
**Pull request by Haywayaheadshot**
### What?

I created a database with PostgreSQL

### Why?

To store information about animals for a veterinary clinic

### How?
- I created the vet_clinic database with PostgreSQL.
- I added an animals table.
- I added data to the animals' table.
- I added queries.

# GOOD DAY.  PLEASE FIND ATTACHED BELOW THE SCREENSHOTS TO THE QUERIES.
1st_query
2nd_query
3rd_query
4th_query
5th_query
6th_query
7th_query
8th_query

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,11 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animals
+(
+    id integer,
```
Comment (Gambit142) [inline schema.sql]: - [ OPTIONAL] You can make the id column autoincrement and not nullable. You can also make it the primary key for this table. The good thing about doing such is that you won't have to manually populate the `id` column with data. 👍 

`id SERIAL PRIMARY KEY` 👍

**Review by Gambit142 — APPROVED** [review]
Hi @Haywayaheadshot 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All codes for creating, structuring and altering tables are in the _**schema.sql**_ file ✔️ 
- All codes that populate tables with data are in the _**data.sql**_ file ✔️ 
- All codes that query or help to analyze data are in the _**queries.sql**_ file ✔️ 

### Optional Suggestions
- Kindly consider the comment tagged `[OPTIONAL]`. Implementing such will only make your code better. 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (Haywayaheadshot) [reply schema.sql]: @Gambit142 Thank you so much I was actually thinking of how to handle that.

**Comment by Haywayaheadshot** [comment]
@Gambit142 Thank you for reviewing my project

**Pull request merged**